shared/src/commonMain/kotlin/org/hyperskill/app/home/presentation/HomeFeature.kt

```diff
@@ -31,18 +39,16 @@ interface HomeFeature {
          * @see ProblemOfDayState
          */
         data class Content(
-            val streak: Streak?,
-            val hypercoinsBalance: Int,
             val problemOfDayState: ProblemOfDayState,
             val repetitionsState: RepetitionsState,
-            val isRefreshing: Boolean = false,
+            internal val isRefreshing: Boolean = false,
```

Comment:
`isRefreshing` flag is needed on Android side, to control whether refresh loaded is shown. Let's leave it from other modules.

Reply:
To observe whether Pull-to-refresh is ongoing you need to look at root Feature's state `isRefreshing` computed property:
```kotlin
data class State(
    val homeState: HomeState,
    val toolbarState: GamificationToolbarFeature.State
) {
    val isRefreshing: Boolean
        get() = homeState is HomeState.Content && homeState.isRefreshing ||
            toolbarState is GamificationToolbarFeature.State.Content && toolbarState.isRefreshing
}
```

This computed property is necessary because we need to finish Pull-to-refresh presentation when both loading operations are finished.